lib/rules/no-restricted-imports.js

```diff
@@ -121,31 +125,67 @@ module.exports = {
         }
 
         /**
+         * Report a restricted path specifically for patterns.
+         * @param {node} node representing the restricted path reference
+         * @returns {void}
+         * @private
+         */
+        function reportPathForPatterns(node) {
```

Comment:
I pulled this method out into a function for consistency with ```reportPath```.  I did not change any logic.